distribution/docs/src/main/resources/_contents/integrating-contents.adoc

```diff
@@ -892,61 +892,72 @@ The response is returned as a data map that contains an internal map with the fo
 [source,json,xml]
 ----
 {
-    "data": {
-        "hits": 1,
-        "metacard-types": {
-            "ddf.metacard": {...}
-        },
-        "id": "857f76c0-da8c-72f5-09cd-536b69e427af",
-        "results": [
-           {
-                "metacard": {
-                    "cached": "2016-06-09T22:46:57.258+0000",
-                    "geometry": {
-                        "coordinates": [
-                            -97.03125,
-                            35.173808
-                        ],
-                        "type": "Point"
-                    },
-                    "type": "Feature",
-                    "actions": [ ... ],
-                    "properties": {
-                        "thumbnail": "..."
-                        "metadata": "<?xml version=\"1.0\" encoding=\"UTF-8\"?><metadata> ... </metadata>",
-                        "resource-size": "92314",
-                        "created": "2016-06-09T22:46:37.707+0000",
-                        "metacard-tags": [
-                            "resource"
-                        ],
-                        "resource-uri": "content:1a83781dd56a4050afc420016d078e22",
-                        "checksum-algorithm": "Adler32",
-                        "metadata-content-type": "image/jpeg",
-                        "metacard-type": "ddf.metacard",
-                        "title": "fixTheBugs_geotagged.jpg",
-                        "resource-download-url": "${secure_url}services/catalog/sources/ddf.distribution/1a83781dd56a4050afc420016d078e22?transform=resource",
-                        "source-id": "ddf.distribution",
-                        "effective": "2016-06-09T22:46:37.707+0000",
-                        "point-of-contact": "",
-                        "checksum": "edfa1836",
-                        "modified": "2016-06-09T22:46:37.707+0000",
-                        "id": "1a83781dd56a4050afc420016d078e22"
-                    }
-                }
+   "data":{
+      "hits":1,
+      "metacard-types":{
+         "ddf.metacard":{...}
+      },
+      "id":"6f0e04e9-acd1-4935-b9dd-c83e770a36d5",
+      "results":[
+         {
+            "metacard":{
+               "internal.local-resource":false,
```

Comment:
@pklinef, @brendan-hofmann, are we all comfortable having an attribute called `internal...` returned to the client here? This is an external API and it feels weird to me to have something called `internal`.